src/components/SearchRefResultsTable.js

```diff
@@ -1,24 +1,25 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Panel } from 'react-bootstrap';
+import { getCsv } from '../actions/ApiActions';
+import apiSearch from '../utils/apiSearch';
 import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
-import { toArray } from '../utils/helperMethods';
-import {CSVLink} from 'react-csv';
 import '../resources/css/react-bootstrap-table-all.min.css';
 
-const SearchRefResultsTable = function ({ results }) {
-  //var size = Object.keys(results).length;
-  //let formattedResults = toArray(results);
-  //formattedResults = formattedResults.slice(0, 5)
+const SearchRefResultsTable = function ({ results, query }) {
   const title = (<h1 style={{ fontSize: '20px' }}>Results</h1>);
   function hideCol(header){
     for (var i in results){
-      if (results[i][header] !== null){
+      if (results[i][header] !== undefined){
         return false
       }
     }
     return true
   }
+  function downloadFile() {
+    const filter = Object.keys(results[0]).toString()
+    apiSearch.getDownload(query,filter)
```

Comment:
You should probably call a redux action which will then call the utils/apiSearch rather than calling it directly